cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -358,6 +358,22 @@ func (dp *DevicePlugin) getDpdkMounts(dpdkDeviceName string) []pluginapi.Mount {
 	}
 }
 
+// heartbeat is only supported on the 4xxx devices. Other devices return "unknown".
```

Comment:
I believe it's available on `c62x, c3xxx,dh895xcc` too? the difference was that on `4xxx` the poll time was fixed.

Reply:
I think you are right. For the `heartbeat/config` it says "This parameter is effective only for c3xxx, c62x, dh895xcc devices."

I'll modify the flow.